config/initializers/create_checks_worker.rb

```diff
@@ -0,0 +1,18 @@
+#Create the workers that read from the queue and create checks.
+sqs_create_checks_url = ENV['AWS_CREATE_CHECKS_SQS_URL'] || ''
```

Comment:
Yeah this is something that Jesus told me about also. I did it because I thought we wanted to migrate to use urls, even if we are not using them yet in other places. I can change it of course but knowing that this functionality is going to be deprecated soon, do you think is worth it?